Fixes related to upcoming chargebyte release with EVerest 2024.3.0

mhei opened the pull request:
This fixes two issues:
- the firmware update via OCPP might fail due to firmware image size
- GetDiagnostics in dual charger setups fails due to incomplete implementation in SystemAggregator module

mhei committed:
remotechargeport: switch to tag 2024.03.1

The release 2024.03.0 included an incomplete commit.
While it compiles, it did not included the complete new desired
functionality.

Signed-off-by: Michael Heimpold <michael.heimpold@chargebyte.com>

mhei committed:
everest-core: add patch for System module

The downloads for Tarragon platform are quite large at the moment,
this we build developer images. The file might exceed the available
space on the /tmp ramfs, so it is better to download to /srv partition
instead.
This commit introduces a patch for this, a clean approach should
be developed with upstream.

Signed-off-by: Michael Heimpold <michael.heimpold@chargebyte.com>

barsnick reviewed (approved):
I didn't review the complete change set in remotechargeport.

We might want to make the System module configurable (upstream) in the long wrong. Fine for now.

Nevertheless, LGTM.

mhei committed:
recipes-backports/date: build as shared library

This helps when using local cross-compiled binaries on the target
which are dynamically linked per default.

This is a temporary fix, upstream PR is pending.

Signed-off-by: Michael Heimpold <michael.heimpold@chargebyte.com>

mhei committed:
Revert "packagegroup-everest: add yq helper tool"

This reverts commit fde1cc36ffa5ad75f2e485c7a82adfb2684a80a0.

Since the tool is written in Go it pulls in the Go runtime
environment and this is too heavy at the moment for our
root filesystem of 1 GB.

Signed-off-by: Michael Heimpold <michael.heimpold@chargebyte.com>
